crowdfunding_compassion/data/demo.xml

```diff
@@ -53,7 +53,7 @@
         <field name="twitter_url">https://twitter.com</field>
         <field name="instagram_url">https://instagram.com</field>
         <field name="personal_web_page_url">https://example.com</field>
-        <field name="product_id" ref="sponsorship_switzerland.product_template_fund_aid"/>
+        <field name="product_id" ref="sponsorship_switzerland.product_template_fund_csp"/>
```

Comment:
The problem you have here in my opinion is that `product_id` references a `product.product` record while `product_template_fund_csp` is a `product.template` record. That's why you get a wrong model. You can replace this like that to fix it:
```suggestion
        <field name="product_id" eval="ref('sponsorship_switzerland.product_template_fund_csp').product_variant_ids[:1].id"/>
```

Reply:
Yes now I have the good product thank you! However the `crowdfunding_impact_text` I added in `products.xml` doesn't seem to be registered on the related `product.product`.

Reply:
Yes it might add the impact text in the template but not in the related products that are already existing. For that to work you would need a migration script. But I think we can leave it like that and configure manually our products once we will put it into production.